### PR TITLE
Homeowork nr 1

### PR DESCRIPTION
**Pull request by ValentineCherniy**
please, check my first homework

### CODE REVIEW

File: homework_nr_1.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+1
```
Comment (gh3rtzz) [inline homework_nr_1.txt]: great, now change it to 2

Comment (ValentineCherniy) [reply homework_nr_1.txt]: Done

Comment (ValentineCherniy) [reply homework_nr_1.txt]: Could you, please, check my updates

**Review by gh3rtzz — APPROVED** [review]
approved

**Pull request merged**